modules/userId/eids.js

```diff
@@ -367,6 +368,18 @@ export function createEidsArray(bidRequestUserId) {
     if (bidRequestUserId.hasOwnProperty(subModuleKey)) {
       if (subModuleKey === 'pubProvidedId') {
         eids = eids.concat(bidRequestUserId['pubProvidedId']);
+      } else if (subModuleKey === 'ftrackId') {
+        // ftrack has multiple IDs so we add each one that exists
+        let eid = {
+          'atype': 1,
+          'id': (bidRequestUserId[subModuleKey]['DeviceID'] || []).join('|'),
+          'ext': {}
+        }
+        for (let id in bidRequestUserId[subModuleKey]) {
+          eid.ext[id] = (bidRequestUserId[subModuleKey][id] || []).join('|');
+        }
+
+        eids.push(eid);
```

Comment:
this block needs tests cases, please

Reply:
no changes to the publisher config implementation, just the eids object interaction is changed
I will also let @ftxmoJason comment on the test cases